endoreg_db/serializers/label_video_segment/_lvs_update.py

```
@@ -0,0 +1,90 @@
+    Label,
+    VideoFile,
+)
+
+import logging
+logger = logging.getLogger(__name__)
+
+if TYPE_CHECKING:
+    from endoreg_db.serializers import LabelVideoSegmentSerializer
+
+
+def _update(self:"LabelVideoSegmentSerializer", instance, validated_data):
```

Pylint (3.3.7)

[refactor] Too many branches (13/12)

(R0912)